### PR TITLE
[FLINK-13092][docs-zh] Translate "Kubernetes" page into Chinese

### PR DESCRIPTION
**Pull request by zm12397**
## What is the purpose of the change

*(For example: This pull request makes task deployment go through the blob server, rather than through RPC. That way we avoid re-transferring them on each deployment (during recovery).)*

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by zentol** [comment]
CI report for commit ad580cc3e675efdd6adaeb89ac0bf6dcf42c2c21: FAILURE [Travis](https://travis-ci.org/dataArtisans/flink/builds/554657763?utm_source=github_status&utm_medium=notification)

File: docs/ops/deployment/kubernetes.zh.md [hunk 1]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
     kubectl create -f jobmanager-deployment.yaml
     kubectl create -f taskmanager-deployment.yaml
 
-You can then access the Flink UI via `kubectl proxy`:
+接下来可以通过`kubectl proxy`命令来访问Flink UI，步骤如下：
 
-1. Run `kubectl proxy` in a terminal
-2. Navigate to [http://localhost:8001/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy](http://localhost:8001/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy) in your browser
+1. 在终端运行`kubectl proxy`命令
+2. 在浏览器中导航到http://localhost:8001/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy页面
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
2. 在浏览器中导航到 [http://localhost:8001/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy](http://localhost:8001/api/v1/namespaces/default/services/flink-jobmanager:ui/proxy) 页面。
```

File: docs/ops/deployment/kubernetes.zh.md [hunk 2]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
 
-In order to terminate the Flink session cluster, use `kubectl`:
+关闭集群时，仍然使用`kubectl`命令，如下所示：
 
     kubectl delete -f jobmanager-deployment.yaml
     kubectl delete -f taskmanager-deployment.yaml
     kubectl delete -f jobmanager-service.yaml
 
-## Flink job cluster on Kubernetes
+## 部署在Kubernetes上的Flink job集群
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
## Kubernetes 上的 Flink job 集群
```

File: docs/ops/deployment/kubernetes.zh.md [hunk 3]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
-  Otherwise Flink components are not able to self reference themselves through a Kubernetes service. 
+  <strong>注意：</strong> 如果使用MiniKube 请确保在部署Flink集群之前先执行 `minikube ssh 'sudo ip link set docker0 promisc on'` ，否则Flink组件不能自动地将自己映射到Kubernetes Service中。
 </div>
 
-## Flink session cluster on Kubernetes
 
-A Flink session cluster is executed as a long-running Kubernetes Deployment. 
-Note that you can run multiple Flink jobs on a session cluster.
-Each job needs to be submitted to the cluster after the cluster has been deployed.
+## 部署在Kubernetes上的Flink session集群
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
## Kubernetes 上的 Flink session 集群
```

File: docs/ops/deployment/kubernetes.zh.md [hunk 4]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
+关闭集群时，仍然使用`kubectl`命令，如下所示：
 
     kubectl delete -f jobmanager-deployment.yaml
     kubectl delete -f taskmanager-deployment.yaml
     kubectl delete -f jobmanager-service.yaml
 
-## Flink job cluster on Kubernetes
+## 部署在Kubernetes上的Flink job集群
+
+Flink job集群是只运行单个flink job的专用集群。
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
Flink job 集群是只运行单个 job 的专用集群。
```

File: docs/ops/deployment/kubernetes.zh.md [hunk 5]
```diff
@@ -175,7 +178,7 @@ spec:
     port: 6125
   - name: ui
     port: 8081
-  selector:
+    selector:
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]: revert this line?

File: docs/ops/deployment/kubernetes.zh.md [hunk 6]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
 </div>
 
-## Flink session cluster on Kubernetes
 
-A Flink session cluster is executed as a long-running Kubernetes Deployment. 
-Note that you can run multiple Flink jobs on a session cluster.
-Each job needs to be submitted to the cluster after the cluster has been deployed.
+## 部署在Kubernetes上的Flink session集群
+
+Flink session集群执行时是作为Kubernetes上长期运行的Deployment。这里值得注意的是，可以在一个session集群上运行多个Flink作业。当然，只有session集群部署好以后才可以在上面提交Flink作业。
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
Flink session 集群是以一种长期运行的 Kubernetes Deployment 形式执行的。这里值得注意的是，可以在一个session集群上运行多个Flink作业。当然，只有session集群部署好以后才可以在上面提交Flink作业。
```

File: docs/ops/deployment/kubernetes.zh.md [hunk 7]
```diff
@@ -28,72 +28,75 @@ This page describes how to deploy a Flink job and session cluster on [Kubernetes
...
 
-A basic Flink session cluster deployment in Kubernetes has three components:
+在Kubernetes上部署一个基本的Flink session集群时，一般包括下面三个组件：
 
-* a Deployment/Job which runs the JobManager
-* a Deployment for a pool of TaskManagers
-* a Service exposing the JobManager's REST and UI ports
+* 运行JobManager的Deployment或者Job
+* 运行TaskManager池的Deployment
```
Comment (wuchong) [inline docs/ops/deployment/kubernetes.zh.md]:
```suggestion
* 运行若干个 TaskManager 的 Deployment
```

**Comment by wuchong** [comment]
Hi @zm12397 , do you have time to update this pull request?

**Comment by klion26** [comment]
@zm12397 Do you still want to work on this PR, if yes, could you please rebase the new master to resolve the conflict

**Comment by wuchong** [comment]
Merged in #16780

**Pull request closed without merging**